jmx-metrics/build.gradle.kts

```diff
@@ -69,6 +69,8 @@ testing {
 
 tasks {
   shadowJar {
+    mergeServiceFiles()
```

Comment:
heads up @jack-berg @jkwatson, I had to add this to make things work with 1.22.0 autoconfigure, since this project pulls in multiple exporter artifacts which each have their own `META-INF/services/io.opentelemetry.sdk.autoconfigure.spi.metrics.ConfigurableMetricExporterProvider` file